ddpui/management/commands/set_uuids_in_orgtask_deployment.py

```diff
@@ -0,0 +1,206 @@
+                continue
+
+            # add uuid to each task config in params
+            params = deployment["parameters"]
+            if "config" in params and "tasks" in params["config"]:
+                for task_config in params["config"]["tasks"]:
+                    if "slug" in task_config:
+                        org_task = OrgTask.objects.filter(
+                            org=org, task__slug=task_config["slug"]
+                        )
+                        if "connection_id" in task_config:
+                            org_task = org_task.filter(
+                                connection_id=task_config["connection_id"]
+                            )
+
+                        org_task = org_task.first()
+
+                        if org_task is not None and org_task.uuid:
+                            task_config["orgtask_uuid"] = str(org_task.uuid)
+                            logger.info(
+                                f"updated task config with uuid for {task_config['slug']}"
+                            )
+                logger.info(
+                    f"updated deployment params for {dataflow.name}|{dataflow.deployment_id}"
+                )
+            else:
+                logger.info(
+                    f"no tasks key found in deployment params for {dataflow.deployment_id}"
+                )
+
+            # update deployment with updated params
+            try:
+                payload = PrefectDataFlowUpdateSchema3(
+                    name=dataflow.name,  # wont be updated
+                    connections=[],  # wont be updated
+                    dbtTransform="ignore",  # wont be updated
+                    cron=dataflow.cron if dataflow.cron else "",
+                    deployment_params=params,
+                )
+                prefect_service.update_dataflow_v1(dataflow.deployment_id, payload)
+                logger.info(
+                    f"updated deployment params for the deployment with id {dataflow.deployment_id}"
+                )
+            except Exception as error:
+                logger.info(
+                    f"Something went wrong in updating the deployment params with id '{dataflow.deployment_id}'"
+                )
+                logger.exception(error)
+                logger.info("skipping to next loop")
+                continue
+
+            # assert if the uuid is present in each task config of each deployment
+            try:
+                deployment = prefect_service.get_deployment(dataflow.deployment_id)
+                params = deployment["parameters"]
+                if "config" in params and "tasks" in params["config"]:
+                    for task_config in params["config"]["tasks"]:
+                        if "orgtask_uuid" not in task_config:
+                            logger.info(
+                                f"uuid not found in task config {task_config['slug']} of deployment with id {dataflow.deployment_id}"
```

Comment:
this should also be an exception right

Reply:
I mean we can but i dont want it to stop from going through other dataflows/deployments. What do you think ?

> this should also be an exception right